ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js

```diff
@@ -42,7 +41,7 @@ export default class ConfirmSeedPhrase extends PureComponent {
 
   componentDidMount () {
     const { seedPhrase = '' } = this.props
-    const shuffledSeedWords = shuffle(seedPhrase.split(' ')) || []
+    const shuffledSeedWords = (seedPhrase.split(' ') || []).sort()
```

Comment:
We might want to rename this to `sortedSeedWords` or similar?

Reply:
Ah, indeed, let me update :)

Reply:
done

Reply:
Thanks! Though there are a couple of instances left of a variable called `shuffledIndex` as well it looks like

Reply:
@Gudahtt let me get those and fix tests

Reply:
@Gudahtt done, I've also updated css classnames

Reply:
Ah right, good catch! Forgot about those